nasa-gibs/worldview
# wv-2670: Modify distraction free mode to display distance scale & minimal timestamp

## Description
Add "minimally distracting timestamp" and scale bar to UI while in distraction free mode.

Fixes # WV-2670

## How To Test
- git checkout wv-2670-kiosk-distraction-mode
- npm run watch
- Reset Worldview to default layers
- Enter "Distraction Free" mode via Info button on the top right
- Confirm that scale is displayed in desktop view
- Confirm date (Year Month Day) is displayed in desktop & mobile views
- Exit Distraction free mode
- Add a geostationary (or otherwise subdaily) layer
- Enter "Distraction Free" mode via Info button on the top right
- Confirm date & time (Year Month Day Hours Minutes) is displayed in desktop & mobile views

@nasa-gibs/worldview
